interface/src/app/mlmodel/mLModelComponents.js

```diff
@@ -0,0 +1,25 @@
+angular.module('adage.mlmodel.components', [
+  'adage.mlmodel.resource'
+])
+
+.component('mLModelSelector', {
+  templateUrl: 'mlmodel/selector.tpl.html',
+  bindings: {
+    selectedMLModel: '='
+  },
+  controller: ['MLModel', '$log', function(MLModel, $log) {
+    var self = this;
+
+    MLModel.get(
```

Comment:
Mmm good point - I guess right now we have a maximum limit of returning 50 objects at a time from Tastypie, but we should generalize it so that all Machine Learning Models come back. I'll put this in as a variable at the beginning of the controller that we can change.